fmn/consumer/producer.py

```diff
@@ -119,6 +119,13 @@ def manage_batch(session, backend, pref):
             log.info("* Found %r queued messages" % len(queued_messages))
 
         for recipient in recipients:
+
+            # If the preference is disabled, then drop all those queued
+            # messages.
+            if not pref.enabled:
+                log.info("* Pref %r inactive.  Dropping messages." % pref)
```

Comment:
Is pref an object with a ``__repr__`` ? (ie, can it be understood when printed?)